docker_images/sklearn/app/pipelines/__init__.py

```diff
@@ -1,3 +1,3 @@
 from app.pipelines.base import Pipeline, PipelineException  # isort:skip
 
-from app.pipelines.tabular_data_pipeline import TabularDataPipeline
+from app.pipelines.tabular_classification import TabularDataPipeline
```

Comment:
Maybe let's rename the class as well

```suggestion
from app.pipelines.tabular_classification import TabularClassificationPipeline
```